classes/orcid/OrcidManager.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+/**
+ * @file classes/orcid/OrcidManager.php
+ *
+ * Copyright (c) 2014-2024 Simon Fraser University
+ * Copyright (c) 2000-2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class OrcidManager
+ *
+ * @brief Manages using ORCID settings at site and journal level
+ */
+
+namespace PKP\orcid;
+
+use APP\author\Author;
+use APP\core\Application;
+use APP\facades\Repo;
+use PKP\config\Config;
+use PKP\context\Context;
+use PKP\core\Core;
+
+class OrcidManager
+{
+    public const ORCID_URL = 'https://orcid.org/';
+    public const ORCID_URL_SANDBOX = 'https://sandbox.orcid.org/';
+    public const ORCID_API_URL_PUBLIC = 'https://pub.orcid.org/';
+    public const ORCID_API_URL_PUBLIC_SANDBOX = 'https://pub.sandbox.orcid.org/';
+    public const ORCID_API_URL_MEMBER = 'https://api.orcid.org/';
+    public const ORCID_API_URL_MEMBER_SANDBOX = 'https://api.sandbox.orcid.org/';
+    public const ORCID_API_VERSION_URL = 'v3.0/';
+
+    public const OAUTH_TOKEN_URL = 'oauth/token';
+
+    public const ORCID_API_SCOPE_PUBLIC = '/authenticate';
+    public const ORCID_API_SCOPE_MEMBER = '/activities/update';
+    public const ORCID_EMPLOYMENTS_URL = 'employments';
+    public const ORCID_PROFILE_URL = 'person';
+    public const ORCID_WORK_URL = 'work';
+    public const ORCID_REVIEW_URL = 'peer-review';
+
+    public const ENABLED = 'orcidEnabled';
```

Comment:
I have them here rather than in the settings form since they are used in both the site settings and app settings form.  Do you think this is an okay place for them or would it be better to have these constants in one of the settings form classes?

Reply:
I think they can stay here. Maybe to just add a comment above?